api/filter/http/http_connection_manager.proto

```diff
@@ -12,29 +12,52 @@ import "google/protobuf/duration.proto";
 import "google/protobuf/struct.proto";
```

Comment:
Hmm, shouldn't HTTP connection manager be a network filter? I think this might have been my bad from a long time ago that it ended up under `filter/http`, could fix it here as well.

Reply:
I commented on this a few times along the way, but we never changed it to go into network. I'm not sure it's worth fixing at this point as it would change the namespace.

Reply:
We can do it in a separate PR, but probably should clean this up for consistency at some point. It doesn't break wire format.

Reply:
I agree.